community-membership.md

```diff
@@ -93,6 +93,27 @@ code reviews and work towards becoming an *approver* for the subproject that
 they are active in.  Acceptance of code contributions requires at least one
 approver in addition to the reviews by *members.*
 
+## Triager
+
+Triagers assist the maintainers and approvers with project management and 
+backlog organization. The specific workflows and triage requirements depend on 
+the project, and are set by the project maintainers.
+
+Defined by: [Triage permissions](https://help.github.com/en/github/setting-up-and-managing-organizations-and-teams/repository-permission-levels-for-an-organization#repository-access-for-each-permission-level)
```

Comment:
this needs to indicate how we declare person to have a triager role. source of truth. I'd suggest CODEOWNERS file under the commented section. So it will be a single file that declares all permission groups for the repository

Reply:
hmm, actually triagers do not have code-related permissions (write or merge), so having triagers listed in CODEOWNERS may be confusing. 

The official way to make someone a Triager would be to add them to the github Triager team for each project. Searching https://github.com/orgs/open-telemetry/teams is how I look up who is officially an approver, maintainer, etc. That should work for Triagers as well.

Also, every SIG should keep an up tp date membership in their README: https://github.com/open-telemetry/opentelemetry-python#contributing

Do you agree? Should I describe team membership in the Triager definition?

Reply:
An important thing here is that the list that is "source of truth" must be defined and must be source controlled. There is no audit in group membership manipulation.

CODEOWNERs is a good option to keep the source of truth. So we are not enforcing whether list is in README or in CONTRIBUTING.md.

If commenting out triagers in CODEOWNERS is too messy, I'm totally happy with README.

Reply:
Ok, I will add language requiring that the source of truth on membership must be checked into git; either as CODEOWNERS or README.

Reply:
README, CONTRIBUTING, and/or CODEOWNERS.

Reply:
the reason I push hard against README containing the contributors' names is that that's not the info that a person new to a repo needs to see right away, they just want to understand how to use the product :)

Reply:
Agreed; if it's in the README it should be at the bottom, not the top. CONTRIBUTING is probably the best place for this info.